statistics/github/contributor-collect

```diff
@@ -243,3 +248,14 @@ sub time_string_to_epoch {
 
     return $epoch;
 }
+
+# date must be YYYYMMDD
+sub time_string_to_last_sunday_epoch {
+    my ($date_string) = @_;
+    my ($year, $month, $day) = unpack "A4A2A2", $date_string;
+
+    my $date = DateTime->new(year => $year, month => $month, day => $day);
+    my $desired_dow = 7;            # Sunday
+    $date->subtract(days => ($date->day_of_week - $desired_dow) % 7);
```

Comment:
Hmmm, isn't that equivalent to `days => $date->day_of_week % 7` in this case?

Reply:
Right, what you suggested is more perl native ;-)

Reply:
Ahh, I misunderstood you. But yes, that is equivalent to what you wrote...